yml/dp02_dc2.yaml

```diff
@@ -7872,6 +7880,7 @@ tables:
 - name: TruthSummary
   '@id': '#TruthSummary'
   description: Summary properties of objects from the DESC DC2 truth catalog, as described in arXiv:2101.04855. Includes the noiseless astrometric and photometric parameters.
+  primaryKey: "#TruthSummary.id_truth_type"
```

Comment:
It's OK to do this now, as in practice this attribute seems to be the one used for JOINs and functions "as if" it were the primary unique key, but...

It's not clear why `id` is not the primary key.  The description says it's unique, but in the `MatchesTruth` table there's a hint that it might not be unique (and so that the `truth_type` has to be included to achieve uniqueness).

Please add a comment to the YAML to note that anyone re-using this table's schema should think again about whether this is the right choice.